code/src/components.js/Counter.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import { useSelector } from 'react-redux'
+import { BottomToDo, HCounter } from './GlobalStyles';
+
+const Counter = () => {
```

Comment:
To get the counter working you need to start the counter at 0. You can try something like this:
`const Counter = () => {
  const todos = useSelector((store) => store.todos.items);
  const completedTodos = todos.filter((todo) => todo.isDone);

  return (
    <div>
      <p>Completed todos: {completedTodos.length}</p>
    </div>
  );
};
`